lib/resources/Account.js

```diff
@@ -1,151 +1,176 @@
-        K._get(K.Account.ACCOUNT_RESOURCE + '/' + id + '/customFields', params, {}, callback);
-    },
-
-    removeCustomFields: function (id, customFields, user, reason, comment, params, callback) {
-        if (typeof params === 'function') {
-            callback = params;
-            params = {};
-        }
-        var options = {user: user, reason: reason, comment: comment};
-        params.customFieldList = customFields.map(function (item) {
-            return item.customFieldId;
-        }).join(',');
-        K._delete(K.Account.ACCOUNT_RESOURCE + '/' + id + '/customFields', params, options, callback);
-    },
-
-    addTags: function (id, tags, user, reason, comment, params, callback) {
-        if (typeof params === 'function') {
-            callback = params;
-            params = {};
-        }
-        var options = {user: user, reason: reason, comment: comment};
-        params.tagList = tags.map(function (item) {
-            return item.tagDefinitionId;
-        }).join(',');
-        K._post(K.Account.ACCOUNT_RESOURCE + '/' + id + '/tags', {}, params, options, callback);
-    },
-
-    getTags: function (id, params, callback) {
-        if (typeof params === 'function') {
-            callback = params;
-            params = {};
-        }
-        K._get(K.Account.ACCOUNT_RESOURCE + '/' + id + '/tags', params, {}, callback);
-    },
-
-    removeTags: function (id, tags, user, reason, comment, params, callback) {
-        if (typeof params === 'function') {
-            callback = params;
-            params = {};
-        }
-        var options = {user: user, reason: reason, comment: comment};
-        params.tagList = tags.map(function (item) {
-            return item.tagDefinitionId;
-        }).join(',');
-        K._delete(K.Account.ACCOUNT_RESOURCE + '/' + id + '/tags', params, options, callback);
-    },
-
-    addEmail: function (id, email, user, reason, comment, params, callback) {
-        if (typeof params === 'function') {
-            callback = params;
-            params = {};
-        }
-        var options = {user: user, reason: reason, comment: comment};
-        email.accountId = id;
-        K._post(K.Account.ACCOUNT_RESOURCE + '/' + id + '/emails', email, params, options, callback);
-    },
-
-    getEmails: function (id, params, callback) {
-        if (typeof params === 'function') {
-            callback = params;
-            params = {};
-        }
-        K._get(K.Account.ACCOUNT_RESOURCE + '/' + id + '/emails', params, {}, callback);
-    },
-
-    removeEmail: function (id, email, user, reason, comment, params, callback) {
-        if (typeof params === 'function') {
-            callback = params;
-            params = {};
-        }
-        var options = {user: user, reason: reason, comment: comment};
-        K._delete(K.Account.ACCOUNT_RESOURCE + '/' + id + '/emails/' + email.email, params, options, callback);
+(function(exports){
+    var BASE_RESOURCE = exports.BASE_RESOURCE;
```

Comment:
Question: would it be easier to read/maintain if these were in an `else` block?

```
if (typeof require !== 'undefined') {
   var _post = ...
} else {
   var _post = ...
}
```

That way, they are only declared once?